DataProcessor/DataProcessor.cpp

```diff
@@ -0,0 +1,43 @@
+#include "DataProcessor.h"
+
+using namespace std;
+
+DataProcessor::DataProcessor(){
+}
+
+string DataProcessor::showWelcomeMessage(){
+	string welcomeMessage;
+	welcomeMessage = "Welcome to BlinkList";
+	return welcomeMessage;
+}
+
+string DataProcessor::addTask(Data task){
+	dataStorage.addTaskList(task); //dataStorage is a DataStorage object initialized in O/C
```

Comment:
i dont know, when i made it she has not made her DataStorage API, so i just wrote it first